sip-ri/src/main/java/gov/nist/javax/sip/header/HeaderFactoryImpl.java

```diff
@@ -1793,9 +1795,24 @@ public ReferencesHeader createReferencesHeader(String callId, String rel) throws
         return retval;
     }
     
-    
 
+    /**
+     * Create and return a references header.
+     * 
+     * @param callId
+     * @return
+     * @throws ParseException
+     */
 
+    
+    public TargetDialogHeader createTargetDialogHeader(String callId) throws ParseException {
+    	        if (callId == null)
+    	            throw new NullPointerException("null arg callId");
```

Comment:
Let's change the message to "callId shouldn't be null or empty"

Reply:
done